FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,59 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			// to prevent memory leak
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data,response))
+
+			default:
+				completion(.failure(Error.connectivity))
```

Comment:
It can be confusing to see that a default case completes with a connectivity error. Also the developer has to keep in mind that `default` actually means `failure`.
There's no extra value in having `default` here and it make the code harder to understand. 
Please replace `default` with `.failure`
```suggestion
			.failure:
				completion(.failure(Error.connectivity))
```